packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1685,7 +1716,7 @@ export class ContainerRuntime extends EventEmitter
             }
 
             const trace = Trace.start();
-            const treeWithStats = await this.summarize(fullTree || safe);
+            const treeWithStats = await this.summarize(fullTree || safe, simple);
```

Comment:
What happens if fullTree === true & simple === true?
Is this valid combination? Will we re-generate summaries for all data stores by using previous summary and appending ops?

Reply:
Fair question. I'll rework the args here since they are confined to ContainerRuntime.

Reply:
Addressed with parameter renames and comments.